Sources/UIKit+Changeset.swift

```diff
@@ -10,16 +10,16 @@ import UIKit
 extension UITableView {
 	
 	/// Performs batch updates on the table view, given the edits of a Changeset, and animates the transition.
-	open func update<T: Equatable>(with edits: [Edit<T>], in section: Int = 0) {
+	open func update<T: Equatable>(with edits: [Edit<T>], in section: Int = 0, with rowAnimation: UITableViewRowAnimation = .automatic) {
```

Comment:
Hmm... I'm just now realizing that this can now be `update(with:, with:)` or `update(with:, in:, with:)`. Seems that the two `with`s are a bit awkward. Let me know if you'd like that changed to `withAnimation:` or something (or just make the change directly if you'd like, of course).